### DISCUSSION TITLE
migrative from VList to Virtualizer not working

### DISCUSSION THREAD

**Discussion by dh049** [inokawa/virtua#411] [Q&A]
on a project i currently am successfully using VList for viewing a messenger type app that adds more messages over time. however, i now need to implement infinite scrolling in both directions, and as soon as i just change from VList to Virtualizer, the scrolling is broken. no errors though. i have followed what i see in the Virtualizer storybook but still not working. any ideas? thanks.

**Comment by dh049** [comment]
the function i pass into onRangeChange fires off once, so its definitely initializing, but the list appears to be unscrollable. calling scrollToBottom does nothing as well.

**Reply by inokawa** [reply]
`VList` is just a wrapper of `Virtualizer`. You can see the actual code to know what's missing.

https://github.com/inokawa/virtua/blob/d64d8744ef4dd001588696e194892bcc162b3541/src/vue/VList.tsx#L81-L104
